llvm/unittests/IR/MetadataTest.cpp

```diff
@@ -2181,7 +2181,7 @@ TEST_F(DICompileUnitTest, get) {
       Context, SourceLanguage, File, Producer, IsOptimized, Flags,
       RuntimeVersion, SplitDebugFilename, EmissionKind, EnumTypes,
       RetainedTypes, GlobalVariables, ImportedEntities, Macros, DWOId, true,
-      false, DICompileUnit::DebugNameTableKind::Default, false, SysRoot, SDK);
+      false, DICompileUnit::DebugNameTableKind::Default, false, SysRoot, SDK, false);
```

Comment:
It looks like you may have forgotten to add the actual test for the new flag below?